FirebaseMessaging/Tests/UnitTests/FIRMessagingTokenOperationsTest.m

```diff
@@ -104,13 +107,10 @@ - (void)setUp {
   _instanceID = @"instanceID";
 
   // `FIRMessagingTokenOperation` uses `FIRInstallations` under the hood to get FIS auth token.
-  // Stub `FIRInstallations` to avoid using a real object.
+  // `FIRMessagingTokenFetchOperation` uses `FIRHeartbeatInfo` to retrieve a heartbeat code.
+  // Stub `FIRInstallations` and `FIRHeartbeatInfo` to avoid using real objects.
```

Comment:
optional: I would comment on each `[self stubInstallations]` and `[self stubHeartbeatInfo]` separately as it looks a bit more readable to me, but I'm not too strong about it.